acceptance/framework/k8s/deploy.go

```diff
@@ -72,6 +73,37 @@ func DeployKustomize(t *testing.T, options *k8s.KubectlOptions, noCleanupOnFailu
 	RunKubectl(t, options, "wait", "--for=condition=available", "--timeout=5m", fmt.Sprintf("deploy/%s", deployment.Name))
 }
 
```

Comment:
Missing comment.

This function is bigger so I would keep it as a full function.